package.json

```diff
@@ -30,18 +31,19 @@
     "order"
   ],
   "dependencies": {
-    "stylelint": "^9.10.1",
-    "stylelint-order": "^2.2.1"
+    "stylelint": "^14.6.1",
```

Comment:
stylelint should be listed as a peerDependency, see https://github.com/stylelint/stylelint/blob/main/docs/developer-guide/plugins.md#peer-dependencies